main.go

```diff
@@ -124,6 +150,7 @@ func makeName(prefix, target string) (string, error) {
 		"Tag":         "${TAG}",
 		"Binary":      "${BINARY}",
 		"ProjectName": "${PROJECT_NAME}",
+		"Amd64":       "",
```

Comment:
could you please provide some extra details how to reproduce it?

the better way is to provide an Option for template

```
//	"missingkey=default" or "missingkey=invalid"
//		The default behavior: Do nothing and continue execution.
//		If printed, the result of the index operation is the string
//		"<no value>".
//	"missingkey=zero"
//		The operation returns the zero value for the map type's element.
//	"missingkey=error"
//		Execution stops immediately with an error.
```

Reply:
```
err = t.Option("missingkey=zero").Execute(&out, varmap)
```

Reply:
you can use the current version to generate the shell for this repository: https://github.com/cxjava/m3u8-downloader, the shell is not correct. 
What I do it just give the idea how to support the [targets](https://github.com/cxjava/m3u8-downloader/blob/main/.goreleaser.yml), sorry don't give more details about it.